patches/uClibc-ng/1.0.12/100-provide-_obstack_free.patch

```diff
@@ -0,0 +1,12 @@
+diff -ur uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c uClibc-ng-1.0.12/libc/misc/gnu/obstack.c
+--- uClibc-ng-1.0.12.orig/libc/misc/gnu/obstack.c	2016-03-13 15:08:44.408962824 -0700
++++ uClibc-ng-1.0.12/libc/misc/gnu/obstack.c	2016-03-13 15:13:30.129322998 -0700
+@@ -385,7 +385,7 @@
+     abort ();
+ }
+ 
+-# if 0
++# if 1
+ /* Older versions of libc used a function _obstack_free intended to be
+    called by non-GCC compilers.  */
+ strong_alias (obstack_free, _obstack_free)
```

Comment:
Actually, disabling obstack in 1.0.12 results in the same kind of link errors (but affecting all obstack symbols, not just _obstack_free) because the _GNU_OBSTACK_INTERFACE in <gnu-versions.h> is declared unconditionally regardless of uclibc-ng having obstack support or not. But I see that has been already fixed in 1.0.13 (which is yet to be added to ct-ng).